Documentation/Contributors/BuildGuide/README.md

```diff
@@ -63,19 +62,8 @@ Cesium uses [npm modules](https://docs.npmjs.com/getting-started/what-is-npm) fo
 npm install
 ```
 
-Once all modules have been installed, run `npm run build` to actually build the code:
-
-```bash
-npm run build
-```
-
-Alternatively, if you want to edit source files and see the changes reflected for testing, use `npm run build-watch`.
-
-```bash
-npm run build-watch
```

Comment:
These should be removed from the `package.json` and `gulpfile.cjs` too.

Reply:
`build-watch` is still a valid task. I want to follow up this PR will removing or renaming outdated tasks.